### PR TITLE
Document Token Url in Using Beyond Identity for Authentication

### PR DESCRIPTION
**Pull request by turnipdabeets**
Add missing token url documentation

### CODE REVIEW

File: docs/using-bi-for-auth.md [hunk 1]
```diff
@@ -62,12 +62,27 @@ A full authorization request url has additional parameters that we need to accou
 https://auth-us.beyondidentity.com/v1/tenants/<tenant_id>/realms/<realm_id>/applications/<application_id>/authorize?
 response_type=code
 &client_id=<client_id_from_application>
-&redirect_uri=<redirect_uri from application>
+&redirect_uri=<redirect_uri_from_application>
 &scope=openid
 &state=<state>
```
[the remark this answers is not in the document]
Comment (turnipdabeets) [reply docs/using-bi-for-auth.md:67]: good idea, updated!

File: docs/using-bi-for-auth.md [hunk 2]
```diff
@@ -62,12 +62,27 @@ A full authorization request url has additional parameters that we need to accou
...
+
+![Screenshot](./screenshots/TokenUrl.jpg)]
+
+The following request parameters can be POSTed as a part of the URL-encoded form values to the API:
+
+```bash
+grant_type=authorization_code&
+&code=<code_return_from_authorization_response>
+&client_id=<client_id_from_application>
+&verifier=<code_verifier_from_PKCE_code_challenge_if_used_in_authorization_request>
```
[the remark this answers is not in the document]
Comment (turnipdabeets) [reply docs/using-bi-for-auth.md]: good catch! updated

File: docs/using-bi-for-auth.md [hunk 3]
```diff
@@ -62,12 +62,29 @@ A full authorization request url has additional parameters that we need to accou
 https://auth-us.beyondidentity.com/v1/tenants/<tenant_id>/realms/<realm_id>/applications/<application_id>/authorize?
 response_type=code
 &client_id=<client_id_from_application>
-&redirect_uri=<redirect_uri from application>
+&redirect_uri=<redirect_uri_from_application>
 &scope=openid
 &state=<state>
+&code_challenge_method<method_from_PKCE_for_public_clients>
+&code_challenge<challenge_from_PKCE_for_public_clients>
```
Comment (Will5) [inline docs/using-bi-for-auth.md:69]: These are not formatting correctly because the '=' is missing 🙂

Comment (turnipdabeets) [reply docs/using-bi-for-auth.md]: thanks!

**Review by Will5 — APPROVED** [review]
LGTM

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**